index 3.js

```diff
@@ -0,0 +1,38 @@
+"use strict"
+
+const now = new Date();
+const nowYear = now.getFullYear();
+const nowMonth = now.getMonth() + 1;
+const nowDay = now.getDate();
```

Comment:
Could you please explain the difference between `getDate` and `getDay`?

Reply:
Yeah sure, getDay() returns the day of the week, for example today is Wednesday and it will return 3, but getDate() returns the day of the month for the specified date. for example it will return 5 for today.